nodebuilder/state/service.go

```diff
@@ -30,13 +32,23 @@ type Module interface {
 	BalanceForAddress(ctx context.Context, addr state.Address) (*state.Balance, error)
 
 	// Transfer sends the given amount of coins from default wallet of the node to the given account address.
-	Transfer(ctx context.Context, to state.AccAddress, amount math.Int, gasLimit uint64) (*state.TxResponse, error)
+	Transfer(
```

Comment:
Was it too long for you to reformat all the methods? :grin:

If you want to make it shorter, you can remove all the names before type params. There are unnecessary for the interfaces.

Reply:
Kinda :D Reverted 

No, I didn't wanted to remove the namings tbh